Documentation/troubleshooting.rst

```diff
@@ -88,7 +89,8 @@ of all nodes in the cluster:
 Logs
 ~~~~
 
-To retrieve log files of a cilium pod, run
+To retrieve log files of a cilium pod, run (replace `cilium-1234` with a pod
+name returned by `kubectl -n kube-system get pods -l k8s-app=cilium`)
```

Comment:
Would making the code snippet:
```
$ CILIUM_POD=$(kubectl -n kube-system get pods -l k8s-app=cilium)
$ kubectl -n kube-system logs --timestamps $CILIUM_POD
```
simplify this? With the added parenthetical the user now has to think about what these commands do but they may not fully understand. I'm assuming the worst-case panic for the troubleshooting, so the simpler the instructions the more likely it will be effective.
It could also be a one-liner, but I'm not sure how well it would format.

Reply:
`kubectl -n kube-system get pods -l k8s-app=cilium` might give you more than one pod

Reply:
hmm, true... maybe it isn't simpler then :/